OrbitQt/GgpClientTests.cpp

```diff
@@ -0,0 +1,73 @@
+
+#include "GgpClient.h"
+
+#include <QCoreApplication>
+#include <QDebug>
+#include <QHostAddress>
+#include <QStringView>
+#include <cassert>
+
+#include "GgpInstance.h"
+
+namespace {
+void checkInstance(const GgpInstance& instance) {
+  assert(instance.pool == "fra-dev-gen1-1080p");
```

Comment:
maybe replace asserts with CHECKS here and below.

Reply:
I actually wrongfully committed this file. Henning wrote this a while ago and I didn't mean to include it. This test hat the prerequisite that the ggp account on the machine has an instance reserved. This is not going to work on the CI, therefore I will remove it. 

I looked into mocking the command line to test this, but I could not find a way to do that with gtest. I will save this test locally until we have an integration test infrastructure and then include it again.

Reply:
ok, sounds good